geomstats/_backend/numpy/linalg.py

```diff
@@ -9,22 +9,21 @@
     eigvalsh,
     inv,
     norm,
-    matrix_rank,
     svd
 )
 
-# TODO(nina): Clean this import
-from geomstats.backend.numpy.__init__ import to_ndarray
+# TODO(nina): Clean this import.
+from geomstats._backend.numpy.__init__ import to_ndarray as _to_ndarray
 
-TOL = 1e-10
+_TOL = 1e-10
 
 
-def is_symmetric(x, tol=TOL):
-    new_x = to_ndarray(x, to_ndim=3)
+def _is_symmetric(x, tol=_TOL):
```

Comment:
This function is also here: https://github.com/geomstats/geomstats/blob/1e88292766310bbccf75bd0a9297b4a8da87f567/geomstats/geometry/matrices.py#L100

However, I think that it is fine to keep both, so that we do not have to instantiate a Matrices objects in the backend?

Reply:
Jepp, definitely agree. The backend should have no knowledge of the `Matrices` class.